src/exporter.py

```diff
@@ -132,6 +133,19 @@ def __init__(
             ["queue_name"],
             registry=self.registry,
         )
+        self.celery_working_process_count = Gauge(
+            f"{metric_prefix}working_process_count",
```

Comment:
Will there be an underscore between metric_prefix and working_process_count?

Reply:
metric_prefix="celery_"